workers/run_tests.sh

```diff
@@ -51,6 +51,7 @@ if [[ -z $tag || $tag == "salmon" ]]; then
     # Download tximport test data
     rm -rf $volume_directory/tximport_test/
     git clone https://github.com/dongbohu/tximport_test.git $volume_directory/tximport_test
+    gunzip $volume_directory/tximport_test/expected_output/gene_lengthScaledTPM.tsv.gz
```

Comment:
I have replaced the gzipped file with unzipped one in my `tximport_test` github repo. So you can remove this line now.